docs/installation.md

```diff
@@ -295,27 +295,17 @@ If you use sbt to install a pre-release, be sure to add the following setting:
 resolvers += Resolver.sonatypeRepo("snapshots")
 ```
 
-### Native image
+### Native binaries
 
-For macOS and Linux, it's possible to download pre-built GraalVm native binaries
-with instant startup and fast performance for short-lived Scalafmt runs.
+It's possible to download pre-built Scala Native binaries
+with instant startup and fast performance for short-lived Scalafmt runs using Scala CLI and 
+[Scala CLI Setup Action](https://github.com/marketplace/actions/scala-cli-setup-action).
 
 ```sh
-VERSION=@STABLE_VERSION@
-INSTALL_LOCATION=/usr/local/bin/scalafmt-native
-curl https://raw.githubusercontent.com/scalameta/scalafmt/master/bin/install-scalafmt-native.sh | \
-  bash -s -- $VERSION $INSTALL_LOCATION
-scalafmt-native --help # should show version @STABLE_VERSION@
+scala-cli format
```

Comment:
This will read the scalafmt version if it exists and download the proper binary. We will need to update it and drop the additional repository, but it should be a smooth transition.